RELEASING.md

```diff
@@ -26,100 +26,210 @@ Don't be scared off by this document! If you already understand the stuff in th
+  5. `additionalConfigureArgs` has correct value, especially in `release-openjdk8-pipeline` it is different than other jdk versions. This requires certain modification manually
+- ensure downstream build jobs in <https://ci.adoptopenjdk.net/job/build-scripts/job/jobs/job/release/job/jobs/jdkXXu/> are created or updated
+  1. `USER_REMOTE_CONFIGS.branch` should get correct release tag name as `releaseTag`
+  2. `BUILD_CONFIGURATION.USE_ADOPT_SHELL_SCRIPTS` is set to `true`
+
+```mermaid
+
+flowchart TD
+
+1["ReleaseChampion"] -- manual run --> 2["release-build-pipeline-generator"] 
+
+2 --generate--> 3.1["release-openjdk8-pipeline"]
+2 --call--> 3.2["release_pipeline_jobs_generator_jdk8u"]
+2 --generate--> 3.3["release-openjdk11-pipeline"]
+2 --call--> 3.4["release_pipeline_jobs_generator_jdk11u"]
+2 --generate--> 3.5["release-openjdk17-pipeline"]
+2 --call--> 3.6["release_pipeline_jobs_generator_jdk17u"]
+2 --generate--> 3.7["release-openjdkXX-pipeline"]
+2 --call--> 3.8["release_pipeline_jobs_generator_jdkXXu"]
+
+3.2-- create --> 3.2.1["jdk8u-release-mac-x64-temurin"]
+3.2-- create --> 3.2.2["jdk8u-release-linux-x64-temurin"]
+3.2-- create --> 3.2.3["jdk8u-release-<os>-<arch>-temurin"]
+
+3.4-- create --> 3.4.1["jdk11u-release-mac-x64-temurin"]
+3.4-- create --> 3.4.2["jdk11u-release-linux-x64-temurin"]
+3.4-- create --> 3.4.3["jdk11u-release-<os>-<arch>-temurin"]
+
+3.6-- create --> 3.6.1["jdk17u-release-mac-x64-temurin"]
+3.6-- create --> 3.6.2["jdk17u-release-linux-x64-temurin"]
+3.6-- create --> 3.6.3["jdk17u-release-<os>-<arch>-temurin"]
+
+3.8-- create --> 3.8.1["jdkXXu-release-mac-x64-temurin"]
+3.8-- create --> 3.8.2["jdkXXu-release-linux-x64-temurin"]
+3.8-- create --> 3.8.3["jdkXXu-release-<os>-<arch>-temurin"]
+
+```
+
+Disable nightly testing so the release builds aren't delayed by any nightly test runs (set `enableTests : false` in [defaults.json](https://github.com/adoptium/ci-jenkins-pipelines/blob/master/pipelines/defaults.json)). Ensure the build pipeline generator job runs successfully (<https://ci.adoptopenjdk.net/job/build-scripts/job/utils/job/build-pipeline-generator/>), and the flag is disabled by bringing up the Build pipeline job and check the  `enableTests` checkbox is unticked.
+
+Add a banner to the website to indicate that the releases are coming in the near future ([Example Changes](https://github.com/adoptium/adoptium.net/blob/main/src/components/Banner.tsx)).
+
+### Steps for every version
+
+### Auto Way
+
+We are in the process of automation release build. Here are the new steps (Switch 19 for your version number):
+
+1. Jenkins "release trigger" job (e.g <https://ci.adoptopenjdk.net/job/build-scripts/job/utils/job/releaseTrigger_jdk19u/>) runs every hour in the release week to check if new GA tag has been detected in the adoptium's source code repo (e.g <https://github.com/adoptium/jdk19u> This excludes https://github.com/adoptium/aarch32-jdk8u)
```

Comment:
Runs every hour during release week, and is it disabled otherwise?  If so, is there a step defined somewhere to ensure the releaseTrigger jobs are enabled?

Reply:
this is set in the jenkins job trigger as `0 H 13-20 1,3,4,7,9,10 *`
Jan, March, April, July, Sep and Oct, run 13th (the weekend before official release for dry-run) - 20th